### DISCUSSION TITLE
有无可能实现“把误输入的拼音字母直接转成汉字”的功能？

### DISCUSSION THREAD

**Discussion by theoractice** [rime/home#821] [Ideas]
> 苹果 iOS 自带的拼音输入法，如果你输入拼音后直接按「确认」键把拼音字母打出来了 …… 别急着删除，双击那些字母，苹果 iOS 输入法可以建议这串拼音对应的汉字。

https://www.chongbuluo.com/thread-8184-1-1.html

对于PC端这个场景也是很常见的（空格错按成回车）。但在我所知范围内，没有任何一个PC端输入法有类似的功能。是技术上无法实现，还是有专利障碍？
请问rime开发者们有研究过这个功能吗？

**Comment by lotem** [comment]
​微軟沒有設置相關接口吧

**Reply by theoractice** [reply]
研究了一下，tsf框架里本身是有接口的，但应用程序尤其是早期的，不一定实现过那些接口。所以结论应该是不可以
